pages/data-visualization/install-and-connect.mdx

```diff
@@ -196,24 +197,67 @@ The desktop application version of Memgraph Lab does not support receiving envir
+| `AUTH_OIDC_OKTA_USER_INFO_URL`           | Okta OIDC user info URL                                                 | `string`   |                    |
+| `AUTH_OIDC_OKTA_CLIENT_ID`               | Okta OIDC client ID                                                     | `string`   |                    |
+| `AUTH_OIDC_OKTA_CLIENT_SECRET`           | Okta OIDC client secret                                                 | `string`   |                    |
+| `AUTH_OIDC_OKTA_CALLBACK_URL`            | Okta OIDC callback URL                                                  | `string`   |                    |
+| `AUTH_OIDC_OKTA_SCOPE`                   | Okta OIDC scope                                                         | `string`   | `"openid profile"` |
+| `AUTH_SAML_ENTRA_ID_IS_ENABLED`          | Enable or disable Entra ID SSO authentication via SAML                  | `boolean`  | `false`            |
+| `AUTH_SAML_ENTRA_ID_DISPLAY_NAME`        | Entra ID SAML display name "Sign in with `<name>`"                      | `string`   | `"Entra ID"`       |
+| `AUTH_SAML_ENTRA_ID_ENTRY_POINT`         | Entra ID SAML entry point                                               | `string`   |                    |
+| `AUTH_SAML_ENTRA_ID_CALLBACK_URL`        | Entra ID SAML callback URL                                              | `string`   |                    |
+| `AUTH_SAML_ENTRA_ID_APP_ID`              | Entra ID SAML application ID                                            | `string`   |                    |
+| `AUTH_SAML_ENTRA_ID_SIGNATURE_ALGORITHM` | Entra ID SAML signature algorithm                                       | `string`   | `"sha256"`         |
+| `AUTH_SAML_OKTA_IS_ENABLED`              | Enable or disable Okta SSO authentication via SAML                      | `boolean`  | `false`            |
+| `AUTH_SAML_OKTA_DISPLAY_NAME`            | Okta SAML display name "Sign in with `<name>`"                          | `string`   | `"Okta"`           |
+| `AUTH_SAML_OKTA_ENTRY_POINT`             | Okta SAML entry point                                                   | `string`   |                    |
+| `AUTH_SAML_OKTA_CALLBACK_URL`            | Okta SAML callback URL                                                  | `string`   |                    |
+| `AUTH_SAML_OKTA_ISSUER`                  | Okta SAML issuer                                                        | `string`   |                    |
+| `AUTH_SAML_OKTA_SIGNATURE_ALGORITHM`     | Okta SAML signature algorithm                                           | `string`   | `"sha256"`         |
+| `ENTERPRISE_LICENSE_ORG_NAME`            | Enterprise license organization name                                    | `string`   |                    |
+| `ENTERPRISE_LICENSE_KEY`                 | Enterprise license key                                                  | `string`   |                    |
```

Comment:
Mention something about "contact our Sales team via XYZ to purchase a key"

Reply:
Actually, I am thinking about adding a link for both ENV variables to https://memgraph.com/docs/database-management/enabling-memgraph-enterprise - it seems better to have one page that describes the enterprise rather than to have "copy" of that description in multiple places. The enterprise link also has a link to request a license or trial.